src/common/utxobased/engine/types.ts

```diff
@@ -48,7 +48,6 @@ export type UtxoSpendInfoOtherParams = ReturnType<
   typeof asUtxoSpendInfoOtherParams
 >
 export const asUtxoSpendInfoOtherParams = asObject({
-  enableRbf: asOptional(asBoolean),
```

Comment:
This is a breaking change. You should leave it in with a `/** @deprecated */` flag.